### PR TITLE
Changes to support Colibri configurations

### PR DESCRIPTION
**Pull request by srinidhih2**

### CODE REVIEW

File: yocto/arm64/colibri-imx8x/metas [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+poky/meta-poky
+meta-openembedded/meta-python
+meta-openembedded/meta-networking
+meta-openembedded/meta-perl
+meta-openembedded/meta-filesystems
+meta-freescale
+meta-freescale-3rdparty
+meta-freescale-distro
+meta-virtualization
+meta-selinux
```
Comment (quitschbo) [inline yocto/arm64/colibri-imx8x/metas]: we just removed that layer, you should also drop meta-selinux here

File: yocto/arm64/colibri-imx8x/metas_ids [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+poky/meta-poky
+meta-openembedded/meta-python
+meta-openembedded/meta-networking
+meta-openembedded/meta-perl
+meta-openembedded/meta-filesystems
+meta-freescale
+meta-freescale-3rdparty
+meta-freescale-distro
+meta-virtualization
+meta-selinux
```
Comment (quitschbo) [inline yocto/arm64/colibri-imx8x/metas_ids]: same as above remove meta-selinux

**Review by quitschbo — CHANGES_REQUESTED** [review]
plz provide signed-off (git commit -s)

File: yocto/arm64/colibri-imx8x/metas (hunk 1 above)
Comment (srinidhih2) [reply yocto/arm64/colibri-imx8x/metas]: Fixed

File: yocto/arm64/colibri-imx8x/metas_ids (hunk 2 above)
Comment (srinidhih2) [reply yocto/arm64/colibri-imx8x/metas_ids]: Fixed

**Comment by quitschbo** [comment]
you can use "git rebase --signoff" to signoff all commits of your PR

**Review by quitschbo — APPROVED** [review]

**Pull request merged**